diff --git a/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/__tests__/quick-sync-events-graph-v2.test.js b/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/__tests__/quick-sync-events-graph-v2.test.js
index 6acc59a..ffafcf9 100644
--- a/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/__tests__/quick-sync-events-graph-v2.test.js
+++ b/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/__tests__/quick-sync-events-graph-v2.test.js
@@ -29,9 +29,9 @@ const EXPECTED_COMMITMENT_GROUP_EVENTS_ARBITRUM = 17000;
 const EXPECTED_NULLIFIER_EVENTS_ARBITRUM = 22000;
 const EXPECTED_UNSHIELD_EVENTS_ARBITRUM = 9000;
 const SEPOLIA_CHAIN = shared_models_1.NETWORK_CONFIG[shared_models_1.NetworkName.EthereumSepolia].chain;
-const EXPECTED_COMMITMENT_GROUP_EVENTS_SEPOLIA = 0;
-const EXPECTED_NULLIFIER_EVENTS_SEPOLIA = 0;
-const EXPECTED_UNSHIELD_EVENTS_SEPOLIA = 0;
+const EXPECTED_COMMITMENT_GROUP_EVENTS_SEPOLIA = 1;
+const EXPECTED_NULLIFIER_EVENTS_SEPOLIA = 1;
+const EXPECTED_UNSHIELD_EVENTS_SEPOLIA = 1;
 const assertContiguousCommitmentEvents = (commitmentEvents, shouldThrow) => {
     let nextTreeNumber = commitmentEvents[0].treeNumber;
     let nextStartPosition = commitmentEvents[0].startPosition;
@@ -127,9 +127,9 @@ describe('quick-sync-events-graph-v2', () => {
         expect(eventLog.commitmentEvents.length).to.be.at.least(EXPECTED_COMMITMENT_GROUP_EVENTS_SEPOLIA);
         expect(eventLog.nullifierEvents.length).to.be.at.least(EXPECTED_NULLIFIER_EVENTS_SEPOLIA);
         expect(eventLog.unshieldEvents.length).to.be.at.least(EXPECTED_UNSHIELD_EVENTS_SEPOLIA);
-        // TODO: Add when there are events
-        // const shouldThrow = true;
-        // assertContiguousCommitmentEvents(eventLog.commitmentEvents, shouldThrow);
+
+        const shouldThrow = true;
+        assertContiguousCommitmentEvents(eventLog.commitmentEvents, shouldThrow);
     }).timeout(90000);
     it('[V2] Should run live Railgun Event Log fetch for Polygon with high starting block', async function run() {
         if (!(0, helper_test_1.isV2Test)()) {
diff --git a/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/quick-sync-events-graph-v2.js b/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/quick-sync-events-graph-v2.js
index 82ae6d2..c375e1b 100644
--- a/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/quick-sync-events-graph-v2.js
+++ b/node_modules/@railgun-community/wallet/dist/services/railgun/quick-sync/V2/quick-sync-events-graph-v2.js
@@ -16,8 +16,7 @@ const sourceNameForNetwork = (networkName) => {
         case shared_models_1.NetworkName.Ethereum:
             return 'ethereum';
         case shared_models_1.NetworkName.EthereumSepolia:
-            // return 'sepolia';
-            throw new Error('No Graph API hosted service for Sepolia');
+            return 'sepolia';
         case shared_models_1.NetworkName.BNBChain:
             return 'bsc';
         case shared_models_1.NetworkName.Polygon:
